code/game/machinery/computer/buildandrepair.dm

```diff
@@ -22,12 +109,49 @@
 	if(state != FRAME_COMPUTER_STATE_EMPTY)
 		balloon_alert(user, "circuit already installed!")
 		return FALSE
-	return ..()
+	if(!anchored && istype(board))
+		balloon_alert(user, "frame must be anchored!")
+		return FALSE
+	. = ..()
+	if(. && !by_hand)
+		state = FRAME_COMPUTER_STATE_BOARD_SECURED
+		update_appearance(UPDATE_ICON_STATE)
```

Comment:
```suggestion
	if(. && !by_hand) // Installing via RPED auto-secures it
		state = FRAME_COMPUTER_STATE_BOARD_SECURED
		update_appearance(UPDATE_ICON_STATE)
	return . 
```
- more dot memes
- Just a comment to explain what's going on, not immediately obvious